staffeln/conductor/backup.py

```diff
@@ -501,8 +497,12 @@ def collect_instance_retention_map(self):
 
         try:
             servers = self.openstacksdk.get_servers(all_projects=True)
-        except OpenstackHttpException:
-            LOG.warn(_("Failed to list servers for all projects."))
+        except OpenstackHttpException as ex:
+            servers = []
```

Comment:
This seems to be a functional change I think here.

Reply:
it's a bug I found. That servers can be left not defined if OpenstackHttpException
As we not looking for raise any error but only warning. it's weird to allow it failed on servers not defined :)